src/test/java/unstable/RepositorySelectorTest.java

```diff
@@ -100,16 +100,39 @@ public void repositorySelectorTest() {
         assertEquals(4, comboBox.getItems().size());
         assertEquals("dummy4/dummy4", primaryRepo);
 
+        // we check if deleting used repo does not remove it
+        click("Repos");
+        push(KeyCode.DOWN);
+        push(KeyCode.RIGHT);
+        push(KeyCode.DOWN);
+        push(KeyCode.DOWN);
+        push(KeyCode.DOWN); // first used repo
+        push(KeyCode.ENTER);
+        PlatformEx.waitOnFxThread();
+        assertEquals(4, comboBox.getItems().size());
+
         // we check if delete repo works
+        click("Boards"); // trigger clicking "Repos"
```

Comment:
With this yes, it works locally. Previously, the test will skip some
commands except the assertion.
On Oct 12, 2015 9:46 AM, "Joshua Lee" <notifications@github.com> wrote:

> In src/test/java/unstable/RepositorySelectorTest.java
> <https://github.com/HubTurbo/HubTurbo/pull/1044#discussion_r41719144>:
>
> >          // we check if delete repo works
> > +        click("Boards"); // trigger clicking "Repos"
>
> So does it work locally?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1044/files#r41719144>.
>